src/mainboard/protectli/vault_ehl/romstage.c

```diff
@@ -1,17 +1,28 @@
 /* SPDX-License-Identifier: GPL-2.0-only */
 
-#include <string.h>
-#include <types.h>
-#include <soc/meminit.h>
+#include <assert.h>
+#include <console/console.h>
+#include <fsp/api.h>
 #include <soc/romstage.h>
+#include <soc/meminit.h>
+#include "gpio.h"
 
+static const struct mb_cfg ddr4_mem_config = {
+	.UserBd = BOARD_TYPE_DESKTOP, /* FIXME */
```

Comment:
```suggestion
	.UserBd = BOARD_TYPE_DESKTOP,
```

Yes, this is the only good option here.